### DISCUSSION TITLE
Unclear meaning / typo in tutor?

### DISCUSSION THREAD

**Discussion by thomasaarholt** [helix-editor/helix#7483] [Q&A]
>  4. Use ( and ) to cycle the primary selection and **remove the
    very second "would"** with Alt-, .

What is meant here? Is there a typo?

https://github.com/helix-editor/helix/blob/d3f8e0592bd577489369e5bd00cddf159f107a24/runtime/tutor#L1025

**Comment by gyreas** [comment]
The 'the very second "would" ' is referring to the second (from the left) "would" in the provided example...
--------------------------v
 --> How much would would a wouldchuck chuck
 --> if a wouldchuck could chuck would?

...so it's not a typo, albeit it's a bit ambiguous

**Reply by thomasaarholt** [reply]
Ah! I see now! I wasn’t familiar with this exact version of the rhyme - as a child I used to say “how much wood would a woodchuck chop if a woodchuck could chop would”.

“remove the second occurrence of” would be a better phrasing here, I think. Shall I make a PR?

**Reply by the-mikedavis** [reply]
Yeah that wording sounds clearer to me, please make a PR 👍
